src/resources/locales/en.json

```diff
@@ -135,11 +138,11 @@
       },
       "menu": {
         "aboutBook": "About this book",
-        "addLib": "Add this book to the librairy",
+        "addLib": "Import the extract",
```

Comment:
Why "extract" and not "book"?

Reply:
This is used only for extracts. The issue us the "addLib" property name which does not reflect that.

Reply:
I see thank you. Shall we leave the l10n key as it is, to avoid modifying consumer code at this stage?

Reply:
Yes, we'll make a tour of badly named i10n properties later, there are several which don't make sense in English.